step1_masking_experiment.py

```diff
@@ -83,16 +84,14 @@
 log.write('trial_number\ttrial_type\t') #the trial number will be logged, no matter which task(s) (if any) is/are chosen
 
 #the welcoming message and log is appended based on which tasks are chosen (if any)
-if task_detec == True:
-	detec_txt = 'Pay attention to whether or not a picture is shown. \n\ ' #this part would be added to the welcome message
+if task_detec: #i.e., "if task_detec == True: "
+	detec_txt = '\n Pay attention to whether or not a picture is shown. ' #this part would be added to the welcome message
```

Comment:
Super nitpicky but you said to check out the style: I deleted the space before the '
```suggestion
	detec_txt = '\n Pay attention to whether or not a picture is shown.' #this part would be added to the welcome message
```